types/react-navigation/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for react-navigation 1.0
 // Project: https://github.com/react-community/react-navigation
-// Definitions by: Huhuanming <https://github.com/huhuanming>, mhcgrq <https://github.com/mhcgrq>
+// Definitions by: Huhuanming <https://github.com/huhuanming>
+//                 mhcgrq <https://github.com/mhcgrq>
+//                 fangpenlin <https://github.com/fangpenlin>
```

Comment:
Could you revert this to the single-line comma-separated format? I think @dt-bot has trouble parsing headers that deviate at all from this exact format.